src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1514,56 +1514,81 @@ private IOperation CreateBoundLocalDeclarationOperation(BoundLocalDeclaration bo
 
             SyntaxNode varStatement;
             SyntaxNode varDeclaration;
+            SyntaxNode varDeclarator;
             switch (kind)
             {
                 case SyntaxKind.LocalDeclarationStatement:
-                {
-                    var statement = (LocalDeclarationStatementSyntax)node;
+                    {
```

Comment:
wrong formatting option?

Reply:
I just assumed that this would be in our editorconfig, and that I wouldn't have had this setting set this way. Apparently that isn't true, reformatted.

---
In reply to: [148084541](https://github.com/dotnet/roslyn/pull/22842#discussion_r148084541) [](ancestors = 148084541)